crates/rpc/rpc-eth-api/Cargo.toml

```diff
@@ -55,4 +55,6 @@ optimism = [
     "reth-primitives/optimism",
     "revm/optimism",
     "reth-provider/optimism",
-]
\ No newline at end of file
+]
+inspectors-js-tracer = ["revm-inspectors/js-tracer"]
+rpc-eth-types-js-tracer = ["reth-rpc-eth-types/js-tracer"]
```

Comment:
should probably also be just `js-tracer`

Reply:
Done in 3e5f678f8431096f36ec7185de038b7a72b2b0c4